pbs/pbsrequest.go

```diff
@@ -229,8 +230,8 @@ func (req PBSRequest) Elapsed() int {
 	return int(time.Since(req.Start) / 1000000)
 }
 
-func (req PBSRequest) GetUserID(BidderCode string) string {
-	if uid, ok := req.UserIDs[BidderCode]; ok {
+func (req PBSRequest) GetUserID(BidderCode family.Name) string {
+	if uid, ok := req.UserIDs[BidderCode.String()]; ok { // TODO: consider moving UserIDs to an array indexed by the FamilyName int.
```

Comment:
gotcha... ok.

The main benefit of having a type at all is to make bugs less likely. Right now, `req.UserIDs[adapter.Name()]` is a really easy mistake to make. It would work _most_ of the time... but not always--which makes it vulnerable to subtle, hard-to-find bugs.

Working in smaller chunks is also a noble goal, though, and I personally tend to make much larger diffs than average (to the point that it annoys people sometimes).

I'm not sure what the answer is, on how to merge well. I do definitely think `familyName` should be pushed around the whole codebase, as far as possible, though.